### DISCUSSION TITLE
##为啥呀？

### DISCUSSION THREAD

**Discussion by DearATRI** [SinKy-Yan/zhenxunbot-docker#19] [Q&A]
![image](https://user-images.githubusercontent.com/95509909/206885540-fae4a44e-f002-4f47-9e57-a55479886d6a.png)
显示数据库连接错误

**Comment by SinKy-Yan** [comment]
看看是不是动过了compose那个配置里关于数据库的东西

**Reply by DearATRI** [reply]
![image](https://user-images.githubusercontent.com/95509909/206886742-1b5810da-0e13-44bd-9073-14b9d732b51f.png)
没有啊

**Reply by SinKy-Yan** [reply]
那就不好说了(´･д･`)我也不清楚，你要不连进postgresql看看那个数据库创建了没有

**Reply by DearATRI** [reply]
好的，谢谢啦
